### PR TITLE
NAS-124716 / 24.04 / Corrected test attribute for ix-table2 columns

### PR DESCRIPTION
**Pull request by RehanY147**

### CODE REVIEW

File: src/app/pages/account/users/user-list/user-list.component.ts [hunk 1]
```diff
@@ -63,7 +63,9 @@ export class UserListComponent implements OnInit {
         .join(', ') || this.translate.instant('N/A'),
       sortable: true,
     }),
-  ]);
+  ], {
+    rowTestId: (row: User) => row.uid.toString(),
```
Comment (undsoft) [inline src/app/pages/account/users/user-list/user-list.component.ts:67]: We should also include what it is in test id, e.g. `user-1`, not just `1`

**Review by undsoft — CHANGES_REQUESTED** [review]
The direction is good. There are some things to improve to make various tables differentiate between themselves.

**Review by denysbutenko — COMMENTED** [review]
CI checks are stopped - I need the latest docker image for testing.

**Review by undsoft — APPROVED** [review]
Looks good, thanks. I'm sure automation engineers will appreciate it.

**Pull request merged**